app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragmentAdapters/CardViewHolder.java

```diff
@@ -1,66 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ListFragmentAdapters;
-
-import android.support.v7.widget.CardView;
-import android.support.v7.widget.RecyclerView;
-import android.view.View;
-import android.widget.ImageButton;
-import android.widget.ImageView;
-import android.widget.TextView;
-
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.R;
-
-/**
- * Created by Kuba10 on 20.08.2017.
- */
-
-public class CardViewHolder extends RecyclerView.ViewHolder {
-
-    public TextView title, description;
-    public ImageView image, dragHandle, showLocation;
-    public CardView placeCardView;
-    public ImageButton favouriteBtn;
-
-    private View view;
-
-    public CardViewHolder(View view) {
-        super(view);
-        this.view = view;
-    }
-
-    public void bindPokePlace(PokePlace place) {
-
-
-        title = (TextView) view.findViewById(R.id.title_text);
-        description = (TextView) view.findViewById(R.id.description_text);
-        image = (ImageView) view.findViewById(R.id.card_image);
-        dragHandle = (ImageView) view.findViewById(R.id.drag_handle);
-        showLocation = (ImageView) view.findViewById(R.id.show_on_map_button);
-        favouriteBtn = (ImageButton) view.findViewById(R.id.favourite_button);
-        placeCardView = (CardView) view.findViewById(R.id.placeCardView);
-
-        dragHandle.setImageResource(R.drawable.ic_drag_handle_black_24dp);
-        showLocation.setImageResource(R.drawable.ic_021_pointer);
-
-        title.setText(place.getTitle());
-        description.setText(place.getDesctription());
-
-        switch (place.getFavourite()) {
```

Comment:
Przerost formy nad treścią. Obiekt albo jest ulubiony albo nie jest. 
1. Czemu trzymasz to jako liczba a nie boolean?
2. Getter dla flag ma inną konwencję, zaczyna się od `is` -> `isFavourite()`
3. `favouriteBtn.setImageResource(place.isFavourite() ? R.drawable.ic_032_star : R.drawable.ic_032_star_empty);` byłoby idealnie

Reply:
Nie do końca wiem jak firebase trzyma bolean - a zera i jedynki jestem pewien, więc tak zostało... Robienie tego rodzaju flag to chyba nie jest zły pomysł?

isFavourite faktycznie skraca mase kodu w kilku miejscach projektu - przy okazji moge pocwiczyc ten wciaz malo intuicyjny zapis